Capstone/Match View/MatchViewController.m

```diff
@@ -120,30 +127,50 @@ - (void)filter {
     }
     NSSortDescriptor *sortingBasedOnDistance = [[NSSortDescriptor alloc] initWithKey:@"distance" ascending:YES];
     self->filteredList = [self->filteredList sortedArrayUsingDescriptors:@[sortingBasedOnDistance]];
-    [self uploadCollegesToParse];
+    [self getCollegesInParse];
+}
+
+- (void)getCollegesInParse {
+    PFQuery *query = [PFQuery queryWithClassName:@"MatchesRelations"];
+    [query includeKey:@"matchedCollege"];
+    [query includeKey:@"author"];
+    [query findObjectsInBackgroundWithBlock:^(NSArray *colleges, NSError *error) {
+        if (colleges != nil) {
+            self->allColleges = colleges;
+            for (MatchesRelations *relations in self->allColleges) {
+                [self->allCollegeFromParse addObject:relations.matchedCollege];
+            }
+        } else {
+            NSLog(@"%@", error.localizedDescription);
+        }
+        [self uploadCollegesToParse];
+    }];
+    
 }
 
-/*
- I wanted to only upload colleges that have all the information that a user would need to
- look up a college. That is why you see a long if statement in the uploadCollegesToParse function. It also
- helps avoid the problem of dealing with NSNull. However, making use of a new Schema in the next PR will
- probably improve it. 
- */
 - (void)uploadCollegesToParse {
```

Comment:
since there's a lot of schema changes involving parse objects/relations, it would be really beneficial to the reviewer to have an explanatory comment at the top of the function that describes what objects are added to each parse variable.